packages/app-harness/.gitignore

```diff
@@ -1,2 +1,11 @@
 
-# Inherit from root
\ No newline at end of file
+# Inherit from root
+
+
+dist
+releases
+.tmp
+build
+
+platformBuilds/harness_tizen
```

Comment:
@ElenaDiachenko what is exact reason to have these ignore bits here? most of them are already covered in root. also if we find some specific extra folders to be ignored should be considered to have them in root as well